ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -41,6 +41,7 @@
 #define NNFW_NEON_BACKEND "acl_neon"
 #define NNFW_SRCN_BACKEND  "srcn"
 #define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND
+#define NNFW_FEATURE_SETINPUTDIM_IMPLEMENTED (0)
```

Comment:
Why is the macro name NNFW_FEATURE_SETINPUTDIM_IMPLEMENTED?

Reply:
Any suggestion?

Reply:
It seems appropriate.